autoptimize.php

```diff
@@ -61,8 +64,15 @@
 	$autoptimize_db_version=$autoptimize_version;
 }
 
+// Load translations from the languages directory.
```

Comment:
This is so that the plugin can use custom translations are put into `wp-content/languages/plugins/` instead of `wp-content/plugins/autoptimize/localization` ;)